src/components/menus/FileUpload.ts

```
@@ -0,0 +1,144 @@
+        if (this.options?.fileUpload?.customMenuInvoke) {
+            this.options.fileUpload.customMenuInvoke((this.editor as InnerEditor).aiEditor);
+        } else {
+            this.fileInput?.click();
+        }
+    }
+    
+    // Listen for theme changes
+    onTransaction(event: any): void {
+        super.onTransaction?.(event);
+        // Check if theme has changed
+        if (event && event.transaction && event.transaction.meta && event.transaction.meta.theme) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)